src/Compilation.zig

```diff
@@ -472,6 +472,11 @@ pub fn create(gpa: *Allocator, options: InitOptions) !*Compilation {
             break :blk false;
         };
 
+        const syslibroot = if (use_lld and options.is_native_os and options.target.isDarwin()) blk: {
```

Comment:
`use_lld` is always false when build_options.have_llvm is false. So if we prefix this with it, we prevent getSDKPath from even being compiled when LLD is a build dependency.

```suggestion
        const syslibroot = if (build_options.have_llvm and use_lld and options.is_native_os and options.target.isDarwin()) blk: {
```

Reply:
Also might make sense to put `std.Target.current.isDarwin()` in there since that must be true for `options.is_native_os and options.target.isDarwin()` to be true.